webportal/src/app/job/job-view/job-table.component.ejs

```diff
@@ -1,38 +1 @@
-<table id="job-table" class="table table-bordered table-hover table-condensed">
```

Comment:
We don't need this file actually. Please merge its content to `job-view.component.ejs`.